### PR TITLE
feat: Adds Medinovita to the list

### PR DESCRIPTION
**Pull request by anjali89r**

### CODE REVIEW

**Comment by nimitbhargava** [comment]
@anjali89r thanks for PR
Can you add your company in the alphabetical order.

**Comment by anjali89r** [comment]
@nimitbhargava done

File: README.md [hunk 1]
```diff
@@ -25,7 +26,8 @@ Name | Twitter | What we do with Vue.js | Contact Person | Open for Event Spaces
 [WIOS Designs](https://wiosdesigns.xyz) | | Custom front-end solutions, games and open source projects | ➖ | ➖ | ➖ 
 [YourStory](https://yourstory.com) | [@YourStoryCo](https://twitter.com/YourStoryCo) | Media Platform | ➖ | ➖ | ➖ 
 [Zoomcar](https://zoomcar.com) | [@ZoomCarIndia](https://twitter.com/ZoomCarIndia) | Self-Driving Car Rental | ➖ | ➖ | ➖ 
-[QArt Fashion](https://www.qart.fashion/) | [@QArtFashion](https://twitter.com/QArtFashion) | For QArt Dashboard | ➖ | ➖ | ➖
+[QArt Fashion](https://www.qart.fashion/) | [@QArtFashion](https://twitter.com/QArtFashion) | For QArt Dashboard | ➖ | ➖ | 
+➖
```
Comment (vinayakkulkarni) [inline README.md]: this _ideally_ shouldn't be in the PR

**Review by vinayakkulkarni — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (anjali89r) [reply README.md]: corrected

**Comment by nimitbhargava** [comment]
@anjali89r Thanks for your contribution 👍

**Pull request merged**